config/routes.rb

```diff
@@ -10,7 +10,7 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
 
-  delete '/logout', to: 'sessions#destroy'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
That's interesting, it's been working for me but I know sometimes it depends what page you're currently on. Was there a particular page you were on before clicking logout that caused the issue? I'll merge it this way and play around with it later today to see if I can get delete to work somehow.